config.example.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  TOKEN: 'YOUR TOKEN HERE'
+};
+
+
+/* Fullstack Review style: */
+/*
+module.exports = {
+  TOKEN: 'YOUR TOKEN HERE'
+};
+*/
+
+/* Chatterbox Client style: */
+/*
+// All this is doing is inserting the parse API keys into every $.ajax
+// request that you make so you don't have to.
+
+// Put your parse application keys here!
+$.ajaxPrefilter(function (settings, _, jqXHR) {
+  jqXHR.setRequestHeader('Authorization', 'GITHUB_API_TOKEN');
+});
+
+// Put your campus prefix here
+window.CAMPUS = 'FILL_ME_IN';
+*/
```

Comment:
If we have everything working with axios, we can probably delete these comments later.